manifests/preflight.yaml

```diff
@@ -4,6 +4,50 @@ metadata:
   name: sentry-enterprise
 spec:
   analyzers:
+    # Fails on preflight page even though registry has been added vendor.replicated.com.
```

Comment:
@aj-jester this analyzer checks for access to the registry directly, but is not aware of our proxy registry. Perhaps our docs need updating here?